code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -64,7 +64,7 @@
 			overmind.add_points(point_rate)
 	health = min(maxhealth, health+core_regen)
 	if(overmind)
-		overmind.update_health()
+		overmind.update_health_hud()
```

Comment:
All mobs now use the same proc to update their health hud icon. (also update_health() is a terrible name for a proc which does NOT update the actual health...)